highlightCode.js

```diff
@@ -1,5 +1,19 @@
 const CodeMirror = require("codemirror/addon/runmode/runmode.node");
 
+const tagsToReplace = {
+   '&': '&amp;',
+   '<': '&lt;',
+   '>': '&gt;'
+};
+
+const replaceTag = function(tag) {
+   return tagsToReplace[tag] || tag;
+}
+
+const safe_tags_replace = function(str) {
+    return str.replace(/[&<>]/g, replaceTag);
```

Comment:
this probably could avoid use of `replaceTag` function and corresponding object with a regex group substitution?

Reply:
But bI wonder if that really makes the code any more clear? The current version is verbose, but quite obvious.